packages/flutter_driver/test/flutter_driver_test.dart

```diff
@@ -1014,7 +1014,7 @@ class FakeVmService extends Fake implements vms.VmService {
   }
 
   @override
-  void dispose() {
+  Future<void> dispose() async {
```

Comment:
I think this code won't cause problems in g3 but it potentially could.

Reply:
well, that is what frob is for :)